Fix so caret and column label do not overlap

carsonwood1 opened the pull request:
This is a fix I believe for the issue where a column label text and the sort caret overlap in RTL mode. Change the CSS class the aligns the caret to be `.tabulator-col-sorter` and not `.tabulator-arrow` with RTL styling.

#3498

olifolkerd commented:
Hey @carsonwood1 

Do you have a link to a JSFiddle that demonstrates the current issue?

Cheers

Oli :)

olifolkerd commented:
How does this compare to the fix proposed by @rowenbuzz on the issue?

Cheers

Oli :)

carsonwood1 commented:
@olifolkerd 

Looks like he changed the position to be initial what the code used to be it appears.

I just updated instead of position right on the `.tabulator-arrow` , add position right to the container of the arrow `.tabulator-col-sorter` . 

It was just what I noticed and fixed looking at the CSS locally, so I wouldn't be offended if y'all went with going back to position: initial instead 👍

olifolkerd commented:
Thanks for the response, i think i will opt for the "initial" approach as it is a little cleaner.

Cheers

Oli :)
